src/util_pyinstaller/README.md

```diff
@@ -0,0 +1,43 @@
+# Packaging ZML with PyInstaller
+
+*By Martin Zak*
+
+This example demonstrates how to package a Python script that uses Zaber Motion Library with PyInstaller.
+The script script itself in `main.py` is a simple motion example.
+
+## PyInstaller
+
+PyInstaller is a tool to package Python scripts into standalone executables.
+When used with Zaber Motion Library the shared library (.dll, .so, or .dylib) file must be explicitly included in the package.
+The command below taken from `build.bat` demonstrates how to include the library.
+
+```ps1
+pyinstaller --onefile --add-binary ".venv\Lib\site-packages\zaber_motion_bindings\zaber-motion-lib-windows-amd64.dll;zaber_motion_bindings" main.py
+```
+
+## Dependencies
+
+- The example is made for Windows, but it can be easily adapted for other platforms.
+- Python 3.10 or newer
+- The script uses `venv` to manage virtual environment and dependencies.
+- The dependencies are listed in the requirements.txt file.
+
+## Building the executable
+
+To build the executable:
+
+```ps1
+.\setup_venv.bat
+.\build.bat
+```
+
+You'll find the distributable executable `main.exe` in the `dist` directory.
+
+## Running the Script
+
+To run the script:
```

Comment:
Suggestion: move this section above "To build the executable", so people don't confuse this section as executing the already compiled executable.  Also, maybe add a section that they should change the COM port in the script before running.

Reply:
Done